content/post/value-reordering-lists.md

```diff
@@ -0,0 +1,238 @@
+what they _need_, or what they _should see_. 
+When recommending news articles, if you're good at
+using "people like you read content like this" collaborative filters, you
+can create [filter bubbles](), where your users consistently see content with their
+particular point of view. Although filter bubbles on a sports web site could keep you
+from seeing articles you might otherwise like,
+filter bubbles in political contexts can [create polarization](). 
+When the mission of your site is to inform people, simply optimizing clicks can
+be solving the wrong problem.
+
+My current company, 
+[WayUp](http://wayup.com), also has to think carefully about what problem
+we're trying to solve. Among other things, WayUp recommends job listings to 
+college students and young professionals, where the goal of the recommendations is to
+get our users to find great job opportunities, without having to spend a lot of time
+searching or scrolling. For us, maximizing total clicks probably means we're creating
+an inefficient experience!
+A pretty good outcome for us is a user who creates a profile, sees some recommendations, 
+clicks on
+the first one, applies, gets the job, and doesn't come back (except to maybe read
+blog posts) for two years. That's one click. So we don't try to maximize total
+clicks. Instead, we look at metrics
+like the proportion of users who apply to any jobs, or the positions in the recommendations
+list of applied-to jobs. 
+
+# Reordering and Optimization
+
+Let's return to that FDNY building-inspection problem to illustrate the next level of
+complexity. It's great that we have a ranked list of
+locations to inspect, in decreasing risk of fire. But what if that list happened
+to be ordered such that locations 1, 3, and 5 were on the far West side of town, and 
+locations
+2, 4, and 6 were on the far East side? We'd be wasting a great deal of time criss-crossing,
+which could put buildings at risk longer than they'd need to be.
+Perhaps it would be better to take travel-time into consideration?
+
+This sort of optimization problem has been tackled for many decades by
+the field of operations research (OR), theoretical computer science, and applied math. 
+A classic example is the 
+[traveling salesperson problem](https://en.wikipedia.org/wiki/Travelling_salesman_problem),
+where the goal is to
+order a list of cities to minimize total travel distance. In the FDNY example, we have
+a 
+[multi-objective optimization](https://en.wikipedia.org/wiki/Multi-objective_optimization) 
+problem, where geographic optimization is combined with a ranking problem. 
+If we put a _cost_ on inspecting locations in the wrong order, and a _cost_ on
+driving, we can numerically optimize the order such that the overall cost is minimized. 
+By using techniques such as 
+[genetic algorithms](https://en.wikipedia.org/wiki/Genetic_algorithm), we might learn that the best approach
+is to look at one neighborhood every day.
+
+# Presenting Ordered Lists
```

Comment:
This last section feels like sort of a hodge-podge. You cover:
* How to use ranking scores outside of list presentation
* Liklihood of outcome vs likelihood of benefit
* Plain lists vs red / yellow / green and score presentations

I think that given the rest of the article, the most relevant idea is likelihood of outcome vs likelihood of benefit. I'd consider scrapping the rest of the content on presentation and product features (or, even better, pulling it into a separate blog post!) and focusing on that.